src/components/forms/SubnetTierForm.js

```diff
@@ -267,6 +330,10 @@ SubnetTierForm.propTypes = {
   enableModal: PropTypes.func,
```

Comment:
there are two hides in this form - one in data and one in props. do we need both?

Reply:
yes, needed for different use cases